samcli/local/services/base_local_service.py

```diff
@@ -96,39 +97,18 @@ def get_lambda_output(stdout_stream):
         -------
         str
             String data containing response from Lambda function
-        str
-            String data containng logs statements, if any.
         bool
             If the response is an error/exception from the container
         """
-        # We only want the last line of stdout, because it's possible that
-        # the function may have written directly to stdout using
-        # System.out.println or similar, before docker-lambda output the result
-        stdout_data = stdout_stream.getvalue().rstrip(b"\n")
-
-        # Usually the output is just one line and contains response as JSON string, but if the Lambda function
-        # wrote anything directly to stdout, there will be additional lines. So just extract the last line as
-        # response and everything else as log output.
-        lambda_response = stdout_data
-        lambda_logs = None
-
-        last_line_position = stdout_data.rfind(b"\n")
-        if last_line_position >= 0:
-            # So there are multiple lines. Separate them out.
-            # Everything but the last line are logs
-            lambda_logs = stdout_data[:last_line_position]
-            # Last line is Lambda response. Make sure to strip() so we get rid of extra whitespaces & newlines around
-            lambda_response = stdout_data[last_line_position:].strip()
-
-        lambda_response = lambda_response.decode("utf-8")
+        lambda_response = stdout_stream.getvalue().decode("utf-8")
```

Comment:
Do we still need the `rstrip(b"\n")` after the `getvalue`? We had it there before but not sure if it is strictly required anymore.

Reply:
I've confirmed this by running local integration tests (some of them confirms the output of the command). So I believe we don't need that particular line